### PR TITLE
add on option to remove the storage connection string from the footer…

### PR DESCRIPTION
**Pull request by sandorfr**
… #1078

### CODE REVIEW

File: src/Hangfire.Core/Dashboard/RazorPage.cs [hunk 1]
```diff
@@ -93,6 +94,7 @@ internal void Assign(DashboardContext context)
             Storage = context.Storage;
             AppPath = context.Options.AppPath;
             StatsPollingInterval = context.Options.StatsPollingInterval;
+            DisplayStorageConnectionString = context.Options.DisplayStorageConnectionString;
```
Comment (pieceofsummer) [inline src/Hangfire.Core/Dashboard/RazorPage.cs]: Should be initialized in another `Assign()` overload too.

Comment (sandorfr) [reply src/Hangfire.Core/Dashboard/RazorPage.cs]: Done ;)

**Comment by pieceofsummer** [comment]
I also think it's no longer reasonable to add an extra page property for each Options property of interest (there's already two of them, and even more after this PR).

Would make sense to add the full `Options` property instead, and dereference its subproperties in the page code.

**Comment by odinserj** [comment]
Thanks guys for PR and for reviewing it. I'm merging these changes, they will be released with 1.6.18 next week, since they are related to security.

**Pull request merged**